wiki/Skinning/tr.md

```diff
@@ -0,0 +1,32 @@
+# Tema yapımı
+
+Tema yapımı osu!'nun olmazsa olmaz özelliklerinden biridir. Bu, oyuncuların orijinal tema yapma elementlerinden kendi temalarını oluşturmalarını sağlar! Temalar eğlence için olmasından, konseptli, PRO, veya oynaması neredeyse imkansıza kadar çeşitlendirilebilir.
```

Comment:
```suggestion
Tema yapımı osu!'nun olmazsa olmaz özelliklerinden biridir. Bu, oyuncuların orijinal tema yapma elementlerinden kendi temalarını oluşturmalarını sağlar! Temalar eğlencesine, konseptli, PRO, veya oynaması neredeyse imkansız olmaya kadar farklılık gösterebilir.```

Reply:
Is there a reason behind the triple backticks you added in your suggestion?